DEV-INTRO.md

```diff
@@ -12,16 +12,37 @@ If running on a linux system it is recommended to add your user to the docker gr
 
 To get started with development in docker we need to bring up the docker stack first. In the main pecan folder you will find the [docker-compose.yml](docker-compose.yml) file that can be used to bring up the pecan stack. There is also the [docker-compose.dev.yaml](docker-compose.dev.yaml) file that adds additional containers, and changes some services to make it easier for development.
 
+By default docker-compose will use the files `docker-compose.yml` and `docker-compose.override.yml`. We will use the default `docker-compose.yml` file from PEcAn. The `docker-compose.override.yml` file can be used to configure it for your specific environment, in our case we will use it to setup the docker environment for development. To do this we simply rename the `docker-compose.dev.yml` file to `docker-compose.override.yml`. You can now use the command `docker-compose` to launch all the containers setup for develoopment.
+
+If you in the past had loaded some of the data, but would like to start from scratch you can simply remove the `volumes` folder (and all the subfolders) and start with the "First time setup" section again.
+
 ### First time setup
 
 The steps in this section only need to be done the fist time you start working with the stack in docker. After this is done you can skip these steps. You can find more detail about the docker commands in the [pecan documentation](https://pecanproject.github.io/pecan-documentation/master/docker-index.html).
 
+Before doing anything it is recommended to make sure you have the lastest docker images ready. You can do a `docker-compose pull` to get the latest images.
+
+* setup .env file
+* create folders to hold the data
+* load the postgresql database
+* load some test data
+* copy all R packages (optional but recommended) 
```

Comment:
Would this actually be optional for anyone working in a dev environment? For example, wouldn't one need to copy the R packages to `volumes` in order to run a workflow in the localhost RStudio?

Reply:
in theory it should work without it, just the problem is that the first time it will have to install ALL dependencies as well, so it will take a lot longer to compile PEcAn.

If you want to just run the workflow, there is no need for the development route as you can use the `docker.sipnet.xml` file and run that in the rstudio container.